src/styles/components/_component-card.scss

```diff
@@ -39,12 +39,15 @@
       font-weight: 700;
       z-index: 1;
       font-size: map.get($font-sizes, "sm");
-      line-height: 4.6667rem; // 75px
+      line-height: map.get($line-heights, "lg"); // 28px/20
       letter-spacing: 0.02em;
+      height: 5.5rem; // 88px/16
       background-color: var(--light-background-color);
       position: relative;
       text-align: center;
-      display: block;
+      display: flex;
+      align-items: center;
+      justify-content: center;
```

Comment:
Is there a reason the `display` values got switched here?

Reply:
Yes- when we initially built this card, the line-height in Figma was 75px, but that has since been changed. We never updated it here, so when I put in the more accurate 28px for this header, it no longer matched the designs. That's why I had to add the `height` and all of the `flex` properties.